torchrl/modules/utils/mappings.py

```diff
@@ -65,7 +65,11 @@ def mappings(key: str) -> Callable:
 
     Args:
         key (str): one of "softplus", "exp", "relu", "expln",
-        or "biased_softplus".
+        or "biased_softplus". If key beggins with "biased_softplus",
+        then the key needs to take the following form:
+        "biased_softplus_{bias}" where bias is a float and the bias term of the biased softplus.
+        or "biased_softplus_{bias}_{min_val}" where bias is a float and the bias term,
```

Comment:
Alternatively, the `"biased_softplus_{bias}_{min_val}"` syntax can be used. In that case, the additional `"min_val"` term is a floating point number that will be used to encode the minimum value of the softplus transform.
In practice, the equation used is `softplus(x + bias) + min_val`, where `bias` and `min_val` are values computed such that the conditions above are met.